src/astro/package.json

```diff
@@ -11,15 +11,10 @@
     "build-css": "tailwindcss build -i src/styles/styles.css -o public/styles/styles.css"
   },
   "dependencies": {
-    "@astrojs/mdx": "^1.1.1",
-    "@astrojs/react": "^3.0.3",
+    "@astrojs/mdx": "^1.0.3",
```

Comment:
You are inadvertently downgrading mdx, here, I think. Run 'git fetch && git merge origin/main' while on this branch.

Reply:
Good catch. Corrected this in the next commit.